vignettes/datatable-benchmarking.Rmd

```diff
@@ -68,27 +68,27 @@ options(datatable.optimize=2L)
 options(datatable.optimize=3L)
 ```
 `options(datatable.optimize=2L)` will turn off optimization of subsets completely, while `options(datatable.optimize=3L)` will switch it back on.
-Those options affects much more optimizations thus should not be used when only control of index is needed. Read more in `?datatable.optimize`.
+Those options affect many more optimizations thus should not be used when only control of index is needed. Read more in `?datatable.optimize`.
 
 # _by reference_ operations
 
-When benchmarking `set*` functions it make sense to measure only first run. Those functions updates data.table by reference thus in subsequent runs they get already processed `data.table` on input.
+When benchmarking `set*` functions it makes sense to measure only first run. Those functions update data.table by reference thus in subsequent runs they get already processed `data.table` on input.
```

Comment:
to measure the first run only

these functions update their input by reference, so subsequent runs will use the already-processed data.table, this biasing the results.